### ISSUE TITLE
Codify

### ISSUE THREAD

**Issue by kris-nova** [kris-nova/naml#40]
I want to add a feature to help users automatically generate `.go` files from existing yaml files. 

### User Experience

I think we should have something like this:

```bash 
cat deployment.yaml | naml codify > deployment.go
kubectl get po <name> -oyaml | naml codify > pod.go
```

Maybe also we can consider directories and more later. For now lets get the 1:1 things working and build out a system with the Go compiler to validate and format the code.

**Comment by kris-nova** [comment]
interested if anyone has any experience reports for `0.2.1`

**Comment by kris-nova** [comment]
closed with #42
